src/SFML/Graphics/Shader.cpp

```diff
@@ -1317,7 +1317,7 @@ bool Shader::isGeometryAvailable()
 
 
 ////////////////////////////////////////////////////////////
-bool Shader::compile(const char* vertexShaderCode, const char* geometryShaderCode, const char* fragmentShaderCode)
+bool Shader::compile(const char* /* vertexShaderCode */, const char* /* geometryShaderCode */, const char* /* fragmentShaderCode */)
```

Comment:
We could use `[[maybe_unused]]` for these instead of comments, but I don't particularly care.

Reply:
I did that but this has to be C++98 compliant (because of 2.6.x) and I had to revert because while gcc and clang ignore unknown attributes sometimes, MSVC throws errors.

Reply:
Oh, sorry -- I didn't realize this was being merged into 2.x.